src/ipc-handlers.ts

```diff
@@ -122,11 +122,10 @@ export async function importSite(
 		};
 		const result = await importBackup( backupFile, site.details, onEvent, defaultImporterOptions );
 		if ( result?.meta?.phpVersion ) {
-			await updateSite( event, {
-				...site.details,
-				phpVersion: result.meta.phpVersion,
-			} );
+			site.details.phpVersion = result.meta.phpVersion;
+			await updateSite( event, site.details );
```

Comment:
I wonder if we could let the caller (in this case the hook `useImportExport`) handle all the updates for the site after the import. This way we won't need to update it in two phases:
1. Here we update the site details in the site server and user data.
2. In `useImportExport` hook, with the returned data, we update the site details in the state of `useSiteDetails` hook.

I think this IPC handler should be only in charge of the import process and be agnostic to what needs to be updated on the site details.

WDYT?

Reply:
Thank you for this suggestion. I think it's wise to re-use the same logic that is already implemented 👍 .
I've removed `updateSiteState` in favor of `updateSite`.